ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -2552,6 +2552,32 @@ def test_tensorflow_MatrixInverse(  # NOQA
     )
 
 
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Rint",
```

Comment:
please add the `aliases` argument here as well. It should be something like `aliases = ['tensorflow.raw_ops.Round']`. Kindly double check, rest the PR seems good to go !

Reply:
@Aarsh2001 done! thanks